README.md

```diff
@@ -1,2 +1,242 @@
+  transactionHash:
+   '0xd327ba43c044b2e0dc9edf583eed4e81f334d890ba5d4f64733b86cb6aadc7fc',
+  transactionIndex: '0x0',
+  blockHash:
+   '0x7345af897aa578d76974ae96715ca39facb8466f11cd4ea32927705533759d14',
+  logIndex: '0x0',
+  removed: false }
+  ```
+
+## Multinode example
+The examle provides a simple interaction with pantheon nodes created in [Pantheon-Quickstart](https://github.com/PegaSysEng/pantheon-quickstart).
+
+1. Deploy `EventEmitter` contract.
+
+data - Binary of the contract
+privateFrom - public Orion key (Node1)
+privateFor -  public Orion key (Node2)
+privateKey - Pantheon private key (Node1), to sign the contract.
+
+```bash
+$ node example/contractCreation.js
+```
+
+2. Get the contract address from the Private Transaction Receipt and set the contract hash into a variable.
+```bash
+$ export CONTRACT_ADDRESS=0xdaa74d5ab2e0b0fad11fca33a76b27c4092f90db
+```
+
+3. Store the value from `node1`.
+```bash
+$ node example/storeValueFromNode1.js
+```
+
+4. Get the value from `node2`
+```bash
+$ node example/getValueFromNode2.js
+```
+
+Sample Output
+```bash
+(node:24545) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
+Transaction Hash 0x9673cb3eebd6cf0139921c74c154bb1eb91c7f3c935362eb19dbdf76a40bbfce
+Waiting ...
+Private Transaction Receipt
+{	jsonrpc: '2.0',
+	id: 1,
+	result:
+		{ contractAddress: null,
+		from: 'Ko2bVqD+nNlNYL5EE7y3IdOnviftjiizpjRt+HTuFBs=',
+		to: 'A1aVtMxLCUHmBVHXoZzzBgPbW/wj5axDpW9X8l91SGo=',
+		output:
+			'0x00000000000000000000000000000000000000000000000000000000000003e8',
+			logs: [] } }
+```
+It can be verified from the output - `0x00000000000000000000000000000000000000000000000000000000000003e8`
+
+5. Store the value from `node2`.
+```bash
+$ node example/multiNodeExample/storeValueFromNode2.js
+```
+
+6. Get the value from `node1`
+```bash
+$ node example/multiNodeExample/getValueFromNode1.js
+```
+
+Sample Output
+```bash
+(node:24545) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
+Transaction Hash 0x9673cb3eebd6cf0139921c74c154bb1eb91c7f3c935362eb19dbdf76a40bbfce
+Waiting ...
+Private Transaction Receipt
+{	jsonrpc: '2.0',
+	id: 1,
+	result:
+		{ contractAddress: null,
+		from: 'A1aVtMxLCUHmBVHXoZzzBgPbW/wj5axDpW9X8l91SGo=',
+		to: 'Ko2bVqD+nNlNYL5EE7y3IdOnviftjiizpjRt+HTuFBs='',
+		output:
+			'0x0000000000000000000000000000000000000000000000000000000000000064',
+			logs: [] } }
+```
+It can be verified from the output - `0x0000000000000000000000000000000000000000000000000000000000000064`
+
+## Run the tests
+To run all the tests:
+```bash
+$ npm test
+```
+
+## How to verify
+
+#### EventEmitter contract
+It can be verified from the output of the last transaction - `0x00000000000000000000000000000000000000000000000000000000000003e8`
```

Comment:
Not quite clear what 'It' refers to here -> is is that the private transaction set the value correctly?